sensum-pwa/docs/setup.md

```diff
@@ -35,4 +35,9 @@ Refs.
 
 - https://github.com/imjoshellis/BGQuickstart.com
 - https://willcodefor.beer/posts/how-to-add-tailwind-to-a-rescript-project/
-- https://vite-plugin-pwa.netlify.app/guide/#setup
\ No newline at end of file
+- https://vite-plugin-pwa.netlify.app/guide/#setup
+
+TODOs:
+
+- https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Add_to_home_screen
```

Comment:
I guess this will help us with Firefox, given currently is not allowing us to add the app to the mobile desktop.